npm-shrinkwrap.json

```diff
@@ -351,9 +351,9 @@
       "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-3.4.1.tgz"
     },
     "bn.js": {
-      "version": "4.11.6",
+      "version": "4.11.5",
```

Comment:
In your reducing of merge conflicts, you might have downgraded a lot of dependencies in our shrinkwrapping? I just upgraded a bunch of our NPM packages.